pages/users/[id].tsx

```diff
@@ -153,35 +212,65 @@ export default function Info({ userData }) {
                     markTshirt();
                   }}
                 >
-                  Mark as Picked Up
+                  Mark T-Shirt Picked Up
                 </motion.button>
               </div>
             )}
-            <p>shirt size: {shirtSize}</p>
-          </div>
-
-          <div className="pb-4">
-            {MLHAcknowledgement ? (
-              <p>
-                MLHAcknowledgement: <span className="text-green-400">True</span>
-              </p>
+            {pickedUpCanes ? (
+              <div className="mt-4 mb-2">
+                <p className="flex">
+                  Canes Pickup:{' '}
+                  <span className="text-green-500 pt-1 pl-1">
+                    <FaCheckCircle />
+                  </span>
+                </p>
+              </div>
             ) : (
-              <p>
-                MLHAcknowledgement: <span className="text-red-400">False</span>
-              </p>
+              <div className="mt-4 mb-2 flex flex-col justify-center items-center">
+                <p className="flex">
+                  Canes Pickup:
+                  <span className="text-red-500 pt-1 pl-1">
+                    <AiFillCloseCircle />
+                  </span>
+                </p>
+                <motion.button
+                  whileHover={{ scale: 1.05 }}
+                  whileTap={{ scale: 0.995 }}
+                  className="flex items-center self-center h-11 px-4 font-semibold text-lg rounded-md bg-purple text-white cursor-pointer"
+                  onClick={() => {
+                    markCanes();
+                  }}
+                >
+                  Mark Canes Picked Up
+                </motion.button>
+              </div>
             )}
           </div>
-          <p className="pb-4">
-            qualified:{' '}
+
+          <p className="flex">
+            Qualified:{' '}
             {qualified == '' ? (
-              <span className="text-red-400">pending</span>
+              <span className="text-red-500 pt-1 pl-1">
+                <AiFillCloseCircle />
+              </span>
             ) : (
-              <span className="text-green-400">qualified</span>
+              <span className="text-green-500 pt-1 pl-1">
+                <FaCheckCircle />
+              </span>
             )}
           </p>
-          {/* add button to check people in  */}
-
-          {/* approve or reject user */}
+          <div className="pb-4 flex">
+            MLHAcknowledgement:{' '}
```

Comment:
`MLHAcknowledgement` isn't really a good indicatorrr.

`MLHAcknowledgement` is set to true if 
<img width="406" alt="image" src="https://user-images.githubusercontent.com/62922309/234700629-c73c7178-5b87-42ba-9566-4fa651c05227.png">

so for those people who opted out of communications, it would show an 'x' which is kinda misleading yknow. 

I think we jus take out the MLHAwk checkmark here as a whole. We already know everyone agreed to the mlh terms. They can't even apply without checking all the mlh boxes since we made it a required field when applying. so its kinda implied that everyone already agreed to mlh if they have a QR code since they were able to apply successfully.